api/src/main/java/org/pmiops/workbench/db/dao/DataSetDao.java

```diff
@@ -16,6 +17,8 @@
 
   List<DbDataset> findByWorkspaceId(long workspaceId);
 
+  Optional<DbDataset> findDbDatasetByNameAndWorkspaceId(String name, long workspaceId);
```

Comment:
Should remove this in favor of findOne(id)

Reply:
Done